includes/classes/class-plugin-manager.php

```diff
@@ -0,0 +1,409 @@
+<?php
+/**
+ * Class PerformanceLab\Plugin_Manager
+ *
+ * @package PerformanceLab
+ * Author: WordPress Performance Team
+ * Author URI: https://make.wordpress.org/performance/
+ * License: GPLv2 or later
+ * License URI: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ */
+
+namespace PerformanceLab;
+
+use WP_Error;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * General purpose class for managing the UI of standalone performance plugins
+ * and rendering of the UI associated therewith.
+ *
+ * @since n.e.x.t
+ * @access public
+ * @ignore
+ */
+class Plugin_Manager {
+
+	/**
+	 * Get info about all the standalone plugins and their status.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array of plugins info.
+	 */
+	public static function get_standalone_plugins() {
```

Comment:
If you keep using the class based approach, this method should be private, as I don't think it's called anywhere outside of this class.

Reply:
Good call thanks @felixarntz . With the older iteration, before I shifted gears to the newer approach, I did call it (from REST route controllers etc). Will update.

Reply:
@felixarntz I've changed this to private for now, but let me think about what it means to not go class based, although I'm curious to hear how strongly you feel about this.

Please see https://github.com/WordPress/performance/pull/864/commits/9b3c413467cc3000755127e76b609f9e358eea6f for the access level change.

Reply:
@felixarntz this class has been dissolved entirely now.